roles/ceph-common/tasks/main.yml

```diff
@@ -93,10 +88,24 @@
   # Hard code this so we will skip the entire file instead of individual tasks (Default isn't Consistent)
   static: False
 
-- include: facts.yml
-- include: ./checks/check_socket.yml
+- name: get ceph version
```

Comment:
This fails on nodes that have roles like "rbd_mirror" since the "ceph" command hasn't been installed yet